### ISSUE TITLE
DOC: install index url missing from high level doc and incorrect docstring:

### ISSUE THREAD

**Issue by Carreau** [pyodide/micropip#116]
https://github.com/pyodide/micropip/blob/e107d892e3d3d969bdaf6ef2d2063ab47f81e19c/micropip/_commands/install.py#L96-L98

state that index "should support json api", as far as I can tell this is not true as we can see that `Transation` will use `query_package`:

https://github.com/pyodide/micropip/blob/e107d892e3d3d969bdaf6ef2d2063ab47f81e19c/micropip/package_index.py#L262-L284

https://github.com/pyodide/micropip/blob/e107d892e3d3d969bdaf6ef2d2063ab47f81e19c/micropip/package_index.py#L80-L88

which _does_ support simple API.

(I have not tried though).

I think mentioning index could also be added in the pyodide documentation itslef in adiition to direct URLS:

https://github.com/pyodide/pyodide/blob/9f49030ef511296574885720282887bea6c6fd54/docs/usage/loading-packages.md#installing-wheels-from-arbitrary-urls

**Comment by ryanking13** [comment]
Thanks. You are right. `micropip` now support Simple APIs (both HTML and JSON). Could you open a PR?

**Comment by Carreau** [comment]
Yes, I can send a pr a bit later. I'll try to do both one here, and on Pyodide to document it.
